src/auro-badge.js

```diff
@@ -48,8 +72,14 @@ class AuroBadge extends LitElement {
   // function that renders the HTML and CSS into  the scope of the component
   render() {
     return html`
-      <div class=${this.cssClass}>
+      <div class="badge" role="button">
```

Comment:
```suggestion
      <div class="badge">
```
We no longer need role="button" since we have an actual `<button>`